providers/os/resources/os.lr

```diff
@@ -1423,6 +1430,38 @@ windows.feature {
   installState int
 }
 
+// Windows Server feature resource
+windows.serverFeature {
```

Comment:
I think we should make name the default and be marked private

Reply:
Fixed.